package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -1804,4 +1820,28 @@ def generate_table(self, output_format=None):
         logger.debug(
             "WBridge: Stored results as table with %(num_records)d entries.",
             vars())
-        self.table = table
+        self.results.table = table
+
+    @property
+    def table(self):
+        wmsg = ("The `table` attribute was deprecated in MDAnalysis 2.0.0 "
+                "and will be removed in MDAnalysis 3.0.0. Please use "
+                "`results.table` instead")
+        warnings.warn(wmsg, DeprecationWarning)
+        return self.results.table
```

Comment:
remove — I don't think that we normally wrap attributes that are to be removed. Or do we, @IAlibay ?

Otherwise you need to store `self.table` in `self._table` and then use the property that you have here.

Reply:
Table will not be made a `results.table`. Instead we want to completely remove it for 3.0.0 (so that users will just use `generate_table()` to create it when they need it.

If you think we should issue a deprecation warning for the use of the attribute then we can do it. (I would then create `self._results` and have the table property just return that so that we don't trigger the warning when we set table in `generate_table()`.) 

Alternatively, we just put in a deprecation in the docs/CHANGELOG and be done.

Reply:
> Alternatively, we just put in a deprecation in the docs/CHANGELOG and be done.
Sounds sensible to me.